src/exercise6.py

```diff
@@ -0,0 +1,14 @@
+# Во входной строке записан текст. Словом считается
+# последовательность непробельных символов идущих подряд,
+# слова разделены одним или большим числом пробелов или
+# символами конца строки. Определите, сколько различных
+# слов содержится в этом тексте.
+
```

Comment:
try to use only split. this task has one-line solution

Reply:
thank you for the review. changes were done

Reply:
> try to use only split. this task has one-line solution

thank you for the review. changes were done